### PR TITLE
feat: add security policy inline with python.org page and similar repos

### PR DESCRIPTION
**Pull request by JacobCoffee**
## What

- Sources a security policy so that users (hopefully) do not create issues on this repo

## Why

- There isn't one
- Clear communication for users to see how to report potential security issues

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+Please read the guidelines on reporting security issues [on the
+official website](https://www.python.org/dev/security/) for
+instructions on how to report a security-related problem to
+the Python team responsibly.
```
Comment (hugovk) [inline SECURITY.md]: Remove some duplication:

```suggestion
Please read the guidelines [on the
official website](https://www.python.org/dev/security/) for
instructions on how to report a security issues to
the Python team responsibly.
```

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+To reach the response team, email `security at python dot org`.
```
Comment (hugovk) [inline SECURITY.md]: To match the obfuscated email link at https://www.python.org/dev/security/

```suggestion
To reach the response team, email 
<a href="mailto:%73%65%63%75%72%69%74%79%40%70%79%74%68%6F%6E%2E%6F%72%67">security
at python dot org</a>.
```

**Review by hugovk — APPROVED** [review]
Thanks!

**Review by eamanu — APPROVED** [review]

**Comment by Mariatta** [comment]
Thank you! Can we add this at the org level instead of at the repo level?

**Comment by JacobCoffee** [comment]
@Mariatta certainly.

https://github.com/python/.github/pull/7 if you want to check that PR.

**Comment by JacobCoffee** [comment]
Closing in favor of https://github.com/python/.github/pull/7

**Pull request closed without merging**